tests/plugins/test_history_at.py

```diff
@@ -0,0 +1,80 @@
+import json
+import re
+import unittest
+import mock
+
+from pyramid import testing
+
+from kinto import main as kinto_main
+from kinto.core.testing import get_user_headers
+
+from .. import support
+from .test_history import HistoryWebTest
+
+
+class HistoryAtViewTest(HistoryWebTest):
+
+    def setUp(self):
+        self.bucket_uri = '/buckets/test'
+        self.app.put(self.bucket_uri, headers=self.headers)
+
+        self.collection_uri = self.bucket_uri + '/collections/col'
+        resp = self.app.put(self.collection_uri, headers=self.headers)
+        self.collection = resp.json['data']
+
+        self.group_uri = self.bucket_uri + '/groups/grp'
+        body = {'data': {'members': ['elle']}}
+        resp = self.app.put_json(self.group_uri, body, headers=self.headers)
+        self.group = resp.json['data']
+
+        self.record_uri = '/buckets/test/collections/col/records/rec'
+        body = {'data': {'foo': 42}}
+        resp = self.app.put_json(self.record_uri, body, headers=self.headers)
+        self.record = resp.json['data']
+
+    def build_version_object_uri(self, object_url):
+        return object_url + '/version'
+
+    def test_version_doesnt_crash_on_wrong_integer(self):
+        self.app.get('/buckets/version/toto', status=404)
```

Comment:
A missing resource return a `404`, if it was an invalid parameter we would return a `400` but here the resource just doesn't exists because no route are matching it.